tooling/nargo_cli/tests/execution_success/simple_2d_array/Nargo.toml

```
@@ -0,0 +1,7 @@
+[package]
+name = "1_mul"
```

```suggestion
name = "simple_2d_array"
```

Not having this match breaks the `rebuild` script.